src/semgrep_agent/semgrep.py

```diff
@@ -37,7 +37,8 @@
 if TYPE_CHECKING:
     from semgrep_agent.semgrep_app import Scan
 
-semgrep = sh.semgrep.bake(_ok_code={0, 1, 2}, _tty_out=False)
+ua_environ = {**os.environ, **{'SEMGREP_USER_AGENT_APPEND': '(Agent)'}}
```

Comment:
```suggestion
ua_environ = {"SEMGREP_USER_AGENT_APPEND": "(Agent)", **os.environ}
```

This should be nicer though anyway. Also allows overriding with an env var visible to semgrep-agent if you really want to.

Reply:
Sorry, I didn't write this comment in time previously 😅